core/domain/topic_domain.py

```diff
@@ -491,6 +491,10 @@ def require_valid_name(cls, name):
         if name == '':
             raise utils.ValidationError('Name field should not be empty')
 
+        if len(name) > 35:
```

Comment:
Please extract these numbers (35, 240) into named constants like MAX_CHARS_IN_TOPIC_NAME and put them somewhere together with the other constants for the Android app. (Perhaps they need to go in constants.js if you're using them in the frontend, too.)

Reply:
Done